test/integration/running/test_running_cluster.py

```diff
@@ -37,45 +37,70 @@ def stop_application(tt_cmd, app_name, workdir, instances):
         assert not os.path.exists(os.path.join(workdir, run_path, app_name, inst, "tarantool.pid"))
 
 
-@pytest.mark.skipif(tarantool_major_version < 3,
-                    reason="skip cluster instances test for Tarantool < 3")
-def test_running_base_functionality(tt_cmd, tmpdir_with_cfg):
-    tmpdir = tmpdir_with_cfg
-    app_name = "small_cluster_app"
+def wait_cluster_started(tt_cmd, workdir, app_name, instances, inst_conf):
+    files = []
+    for inst in instances:
+        conf = inst_conf(workdir, app_name, inst)
+        files.append(conf['pid_file'])
+        files.append(conf['console_socket'])
+    assert wait_files(5, files)
+
+    def are_all_box_statuses_running():
+        status_cmd = [tt_cmd, "status", app_name]
+        status_rc, status_out = run_command_and_get_output(status_cmd, cwd=workdir)
+        assert status_rc == 0
+        status_info = extract_status(status_out)
+        for inst in instances:
+            inst_id = app_name + ":" + inst
+            if "BOX" not in status_info[inst_id] or status_info[inst_id]["BOX"] != "running":
```

Comment:
Just a cosmetic suggestion. It could have been made a bit clearer, without double checks.
```suggestion
            if status_info[inst_id].get("BOX") != "running":
```

Reply:
Done.